### DISCUSSION TITLE
Connect audit on MongoDB database and use MySQL for models

### DISCUSSION THREAD

**Discussion by jubag-93** [owen-it/laravel-auditing#830] [Q&A] [answered]
Hello, I want to connet my audit with a mongoDB database, but when I config the database connection en config/audit.php I have an error about PDO of php who what excute prepare() sentence. I think this problem is fot the hybrid connection, How I can connect the mongo database using mysql for the models. 

thank you people!

**Comment by parallels999** [comment, marked as answer]
Duplicate of https://github.com/owen-it/laravel-auditing/issues/823

**Reply by jubag-93** [reply]
This don't work, I was tested. And also my models implements for eloquent, no from mongodb,

**Comment by jubag-93** [comment]
I did work! I made a new implementation that connect the model with mongoDB
